docs/ldml/tr35-keyboards.md

```diff
@@ -1163,6 +1167,119 @@ This attribute may be escaped with `\u` notation, see [Escaping](#Escaping).
 
 * * *
 
+### <a name="Element_forms" href="#Element_forms">Element: forms</a>
+
+This element represents a set of `form` elements which define the layout of a particular hardware form.
+
+
+> <small>
+>
+> Parents: [keyboard](#Element_keyboard)
+>
+> Children: [import](#Element_import), [form](#Element_form), [_special_](tr35.md#special)
+>
+> Occurrence: optional, single
+>
+> </small>
+
+***Syntax***
+
+```xml
+<forms>
+    <form id="iso">
+        <!-- ... -->
+    </form>
+    <form id="us">
+        <!-- ... -->
+    </form>
+</forms>
+```
+
+* * *
+
+### <a name="Element_form" href="#Element_form">Element: form</a>
+
+This element represents a specific `form` element which defines the layout of a particular hardware form.
+
+> *Note:* Most keyboards will not need to use this element directly, and the CLDR repository will not accept keyboards which define a custom `form` element.  This element is provided for two reasons:
+
+1. To document the standard hardware layouts used with CLDR for implementations. Implementations can verify the layout, and validate keyboards against the number of items per row.
+
+2. To allow a way to customize the scancode layout for keyboards not intended to be included in the common CLDR repository.
+
+See [Implied Form Values](#implied-form-values), below.
+
+> <small>
+>
+> Parents: [forms](#Element_forms)
+>
+> Children: [scanCodes](#Element_scanCodes), [_special_](tr35.md#special)
+>
+> Occurrence: optional, multiple
+>
+> </small>
+
+_Attribute:_ `id` (required)
+
+> This attribute specifies the form id. The value may not be `touch`.
+
+***Syntax***
+
+```xml
+<form id="us">
+    <scanCodes codes="00 01 02"/>
+    <scanCodes codes="03 04 05"/>
+</form>
+```
+
+##### Implied Form Values
+
+There is an implied set of `<form>` elements corresponding to the default forms, thus implementations must behave as if there was the following import statement:
+
+```xml
+<keyboard>
+    <forms>
+        <import base="cldr" path="techpreview/scanCodes-implied.xml" /> <!-- the version will match the current conformsTo of the file -->
+    </forms>
+</keyboard>
+```
+
+Here is a summary of the implied form elements. Keyboards included in the CLDR Repository must only use these `form=` values and may not override the scanCodes.
```

Comment:
Do you want to mention the actual standards here? (ANSI INCITS 154-1998 (R1999), ISO/IEC 9995-2:2009 etc.)